voicevox_engine/tts_pipeline/tts_engine.py

```diff
@@ -174,6 +174,23 @@ def apply_pitch_scale(moras: list[Mora], query: AudioQuery) -> list[Mora]:
     return moras
 
 
+def apply_pause_length(moras: list[Mora], query: AudioQuery) -> list[Mora]:
+    """モーラ系列へ音声合成用のクエリがもつ無音時間(絶対値)（`pauseLength`）を適用する"""
+    if query.pauseLength is not None:
+        for mora in moras:
+            if mora.text == "、":
+                mora.vowel_length = query.pauseLength
+    return moras
+
+
+def apply_pause_length_scale(moras: list[Mora], query: AudioQuery) -> list[Mora]:
+    """モーラ系列へ音声合成用のクエリがもつ無音時間(倍率)（`pauseLengthScale`）を適用する"""
```

Comment:
こちらも他のドキュメントと合わせてスケールで良さそう！

```suggestion
    """モーラ系列へ音声合成用のクエリがもつ無音時間スケール（`pauseLengthScale`）を適用する"""
```